### PR TITLE
update addons with cpu architecture support indicator

### PR DESCRIPTION
**Pull request by ratnopamc**
*Issue #, if available:*
#591 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/addons/index.md [hunk 1]
```diff
@@ -8,59 +8,59 @@ Within the context of the `eks-blueprints` framework, an add-on is abstracted as
...
-| [`OpaGatekeeperAddOn`](./opa-gatekeeper.md)                            | Adds OPA Gatekeeper                                                                                                                    |
+| [`NginxAddOn`](./nginx.md)                                             | Adds NGINX ingress controller  | ✅ | ✅ |                                                                                                        |
+| [`OpaGatekeeperAddOn`](./opa-gatekeeper.md)                            | Adds OPA Gatekeeper                                                                                                                    | ✅ | ✅ |
 | [`PixieAddOn`](./pixie.md)                                             | Adds [Pixie](https://px.dev) to the EKS Cluster. Pixie provides auto-telemetry for requests, metrics, application profiles, and more.  |
 | [`PrometheusNodeExporterAddOn`](./prometheus-node-exporter.md)      | Adds [prometheus-node-exporter](https://github.com/prometheus/node_exporter) to the EKS Cluster. Prometheus Node Exporter enables you to measure various machine resources such as memory, disk and CPU utilization.  |
```
Comment (elamaran11) [inline docs/addons/index.md:55]: This is ARM64 supportive isnt it?

File: docs/addons/index.md [hunk 2]
```diff
@@ -8,59 +8,59 @@ Within the context of the `eks-blueprints` framework, an add-on is abstracted as
...
+| [`KarpenterAddOn`](./karpenter.md)                                     | Adds [Karpenter](https://github.com/awslabs/karpenter) support for Amazon EKS.                                                         | ✅ | ✅ |
+| [`KubeProxyAddOn`](./kube-proxy.md)                                    | Adds kube-proxy Amazon EKS add-on. Kube-proxy maintains network rules on each Amazon EC2 node.                                         | ✅ | ✅ |
+| [`KubeStateMetricsAddOn`](./kube-state-metrics.md)                  | Adds [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) into the EKS cluster.                                          | ✅ | ✅ |
+| [`MetricsServerAddOn`](./metrics-server)                               | Adds metrics server (pre-req for HPA and other monitoring tools).                                                                      | ✅ | ✅ |
 | [`NewRelicAddOn`](./newrelic.md)                                       | Adds [New Relic](https://newrelic.com/) and [Pixie](https://pixielabs.ai/) observability for Amazon EKS.                               |
```
Comment (elamaran11) [inline docs/addons/index.md:51]: Why is even x86 empty for this addon and few other addons. If the Addon is in this repo and runs  e2e means its atleast x86 compliant.

**Review by elamaran11 — COMMENTED** [review]
@ratnopamc Thankyou for the time to put this PR to work. I would like to see the following :
1. Is see 17 addons are not having empty for both `x86` and `ARM64`. If the addon is in this repo and runs e2e, it means it is `x86` validated.  I would recommend running e2e again with all addons in x86 and put on a x86 check mark for all addons if Pods come clean.
2. I would also recommend to complete the test for all remaining addons for ARM64 too and say its compliant or not.
3. Any of the addon not ARM64 compliant please (X) vs leaving it blank. This way this document PR comes to completion.

Please reachout for any questions. Once again great work.

**Comment by zjaco13** [comment]
Hey @ratnopamc, currently working on a pattern for graviton processors, and also testing out the addons to see what works on a graviton cluster.  Please let me know if you want some help checking more addons that work to the docs

**Comment by ratnopamc** [comment]
@elamaran11 , all comments are incorporated in https://github.com/aws-quickstart/cdk-eks-blueprints/pull/729. Closing this.

**Pull request closed without merging**